packages/webapp/__tests__/ProfilePage.tsx

```diff
@@ -360,3 +379,14 @@ it('should show the reading rank history of the user', async () => {
     }),
   );
 });
+
+it('should show the top reading tags of the user', async () => {
+  renderComponent();
+  await waitForNock();
+  await Promise.all(
+    defaultTopTags.map(async ({ value: tag }) => {
+      const el = await screen.findByText(`#${tag}`);
+      expect(el).toBeInTheDocument();
+    }),
```

Comment:
Not breaking, but we could include a check to see if the bar has a width property of {x}px?

Reply:
Why do we want to check the width?

Reply:
Oh, okay. The percentage. Good point. Let me add that in the specific component's test suite.